modules/shadertools/src/modules/pbr/pbr.js

```diff
@@ -1,4 +1,5 @@
 import project2 from '../project2/project2';
+import lighting from '../lighting/lighting';
```

Comment:
change lighting to pbr-lighting? for the lighting module, all the lighting shaders are sharing the same light sources, for each lighting algorithm, we create a material class and shader modules. Better to have a PbrMaterial class and pbr-lighting shahder module instead of using 'lighting' which is more general

Reply:
The lighting module will only provide the **uniforms** but not any actual code.